framework/source/class/qx/theme/manager/Decoration.js

```diff
@@ -122,8 +122,11 @@ qx.Class.define("qx.theme.manager.Decoration",
       // create and add a CSS rule
       var css = "";
       var styles = instance.getStyles(true);
-      for (var key in styles) {
-
+      
+      // Sort the styles so that more specific styles come after the group styles, 
+      //  eg background-color comes after background
+      var t = this;
+      Object.keys(styles).sort().forEach(function(key) {
```

Comment:
That order is for the new Map type. Object iteration order is unspecified. See http://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order